### PR TITLE
WT-13362 Enable wt_verify.py to accept output of wt tool as an input

### PR DESCRIPTION
**Pull request by zunyiliu**
Add new argument -i/--input_file so wt_verify can accpet output of wt verify command as an input.

### CODE REVIEW

File: tools/wt_verify/wt_verify.py [hunk 1]
```diff
@@ -231,11 +231,11 @@ def parse_dump_blocks():
 
     return data
 
-def parse_dump_pages():
+def parse_dump_pages(input_file: str):
     """
     Parse the output file of dump_pages
```
Comment (etienneptl) [inline tools/wt_verify/wt_verify.py:236]:
```suggestion
    Parse the output file of dump_pages.
```

File: tools/wt_verify/wt_verify.py [hunk 2]
```diff
@@ -578,29 +578,39 @@ def construct_command(args):
 def main():
     parser = argparse.ArgumentParser(description="Script to run the WiredTiger verify command with specified options.")
     parser.add_argument('-d', '--dump', required=True, choices=['dump_blocks','dump_pages'], help='Option to specify dump_pages configuration.')
```
Comment (etienneptl) [inline tools/wt_verify/wt_verify.py:580]:
```suggestion
    parser.add_argument('-d', '--dump', required=True, choices=['dump_blocks','dump_pages'], help='wt verify configuration options.')
```

File: tools/wt_verify/wt_verify.py [hunk 3]
```diff
@@ -578,29 +578,39 @@ def construct_command(args):
...
 
-    if "dump_pages" in command:
-        parsed_data = parse_dump_pages()
+    if "dump_pages" in args.dump:
+        parsed_data = parse_dump_pages(input_file)
     else:
-        # If it's "dump_blocks" then do parse_dump_blocks()
-        parsed_data = parse_dump_blocks()
+        if not "dump_blocks" in args.dump:
+            raise Exception("dump_blocks expected in args.dump")
```
Comment (etienneptl) [inline tools/wt_verify/wt_verify.py:612]:
```suggestion
            raise Exception(f"dump_blocks not found in {args.dump}")
```

**Review by etienneptl — APPROVED** [review]
Thank you! LGTM

**Review by korteland — APPROVED** [review]
LGTM!

**Pull request merged**